helm-util/helm.libsonnet

```diff
@@ -0,0 +1,57 @@
+local d = import 'github.com/sh0rez/docsonnet/doc-util';
+{
+  local this = self,
+
+  '#': d.pkg(
+    name='helm-util',
+    url='github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet',
+    help='`helm-util` provides utilities for using helm in jsonnet',
+  ),
+
+  '#helmTemplate': d.fn(
+    '`helmTemplate` wraps the helmTemplate native function and returns a rendered chart as a map of kubernetes manifests',
+    [
+      d.arg('name', d.T.string),
+      d.arg('chart', d.T.string),
+      d.arg('conf', d.T.object),
+      d.arg('labels', d.T.object),
+    ]
+  ),
+  helmTemplate(name, chart, conf={}, labels={})::
+    this.patchLabels(
+      std.native('helmTemplate')(name, chart, conf),
+      labels { 'app.kubernetes.io/managed-by': 'Helmraiser' }
```

Comment:
Do we need to set this? Tanka does not do so usually, because it let's kubectl do it's thing, being transparent to the cluster.

Can't think of reasons Helmraiser resources should be special in any way

Reply:
It is part of the standard labels (https://helm.sh/docs/chart_best_practices/labels/#standard-labels), I just want to make clear this is not managed-by: helm, because it is not.

Reply:
Not having this label at all should be a good start. If you wan't more clearness, Tanka should label everything it manages.

Reply:
I've moved this up into a variable that can be overwritten easily.